src/UI/Settings/Indexers/Options/IndexerOptionsViewTemplate.hbs

```diff
@@ -25,6 +25,18 @@
         </div>
     </div>
 
+    <div class="form-group">
```

Comment:
Maybe make it advanced? People should be using the normal quality definition size sliders.

Reply:
Yea, good call this is mainly just to set a high end limit as last case, if we had adjustable top end on quality sliders instead of jumping straight to unlimited we probably could just ditch this.